### PR TITLE
[ENG-1291] feat: buy action

### PR DESCRIPTION
**Pull request by artmanque**
Dependencies:

1. https://github.com/API-market/oreid-service/pull/2231

### CODE REVIEW

File: src/core/models.ts [hunk 1]
```diff
@@ -99,11 +99,6 @@ export type SignStringParams = {
   message: string
 }
 
-export type BuyParams = {
```
Comment (yardz) [inline src/core/models.ts]: Why did you remove this?

**Review by traylewin — APPROVED** [review]

**Review by traylewin — APPROVED** [review]
